content/departments/engineering/dev/tools/grpc_cs_enablement.md

```diff
@@ -0,0 +1,165 @@
+# gRPC CS Enablement Guide / Primer
+
```

Comment:
How about a 1-2 line intro?

```suggestion
In 5.2 we're rolling out gRPC as a new mechanism for our internal services to communicate with each other.

No customer should notice that something has changed.

This document aims to provide background information on gRPC & how we use it and provide help in case a customer _does_ notice.
```